cmd/image/qcow2ova/gzip.go

```diff
@@ -0,0 +1,77 @@
+package qcow2ova
+
+import (
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	gzip "github.com/klauspost/pgzip"
+)
+
+// gzip the source file to target
+func gzipit(source, target string) error {
+	reader, err := os.Open(source)
+	if err != nil {
+		return err
+	}
+
+	filename := filepath.Base(source)
+	writer, err := os.Create(target)
+	if err != nil {
+		return err
+	}
+	defer writer.Close()
+
+	archiver := gzip.NewWriter(writer)
+	defer archiver.Close()
+	archiver.Name = filename
+
+	_, err = io.Copy(archiver, reader)
+	return err
+}
+
+// gunzip the source file to target
+func gunzip(source, target string) error {
```

Comment:
gunzipIt? 
Also on the variable naming, should there be consistency ? (src/dest, source/target ??)